contessa/consistency_checker.py

```diff
@@ -74,6 +75,46 @@ def run(
         self.right_conn.ensure_table(quality_check_class.__table__)
         self.upsert(quality_check_class, result)
 
+    def run_only(
```

Comment:
I understand that this is a quick workaround to make it possible to return the results only, but wouldnt it be even easier to just use the result table parameter to make the `run` method decide what to do? E.g. it could return the result in every case, and only do db writes if meaningful result table param is sent in? 
It looks like `run` and `run_only` methods are pretty much almost identical, very easily parametrizable with and if or two, wdyt?

Reply:
There is a difference in objects created. `run` creates sqlalchemy objects and run_only creates plain objects of type CheckResult. Therefore I decided to do it like this instead.